pkg/adapter/mtping/adapter.go

```diff
@@ -91,6 +92,20 @@ func GetNoShutDownAfterValue() int {
 	return 55 // seems a reasonable default
 }
 
+func GetSinkWaitDuration() int {
```

Comment:
is this used?

Reply:
Same concern as mine, it seems unused but the one is similar to the one in cloudevents.go. Please move the other one in v2/config.go